engine/Dockerfile

```diff
@@ -19,9 +19,12 @@ RUN pip install -r requirements.txt
 # https://stackoverflow.com/questions/34398632/docker-how-to-run-pip-requirements-txt-only-if-there-was-a-change/34399661#34399661
 COPY ./ ./
 
-# Collect static files and create an SQLite database
-RUN mkdir -p /var/lib/oncall
+# Collect static files
 RUN DJANGO_SETTINGS_MODULE=settings.prod_without_db DATABASE_TYPE=sqlite3 DATABASE_NAME=/var/lib/oncall/oncall.db SECRET_KEY="ThEmUsTSecretKEYforBUILDstage123" SILK_PROFILER_ENABLED="True" python manage.py collectstatic --no-input
```

Comment:
Before removing [this line](https://github.com/grafana/oncall/pull/1917/files#diff-4f09c06cda07f53f96a39ae416e18ce7abc654598ef625f21b05b92758cdce1bL24), running `python manage.py collectstatic` was creating a DB file and everything worked as expected. After removing this line, the DB file is now created by a Celery worker which runs on a different user than uWSGI, which leads to a permission issue.